gatsby-config.js

```diff
@@ -117,6 +117,20 @@ module.exports = {
             },
           },
         },
+        tessen: {
+          product: 'DOC',
+          subproduct: 'TDOC',
+          segmentWriteKey: 'AEfP8c1VSuFxhMdk3jYFQrYQV9sHbUXx',
+          trackPageViews: true,
+          pageView: {
+            name: 'pageView',
+            category: 'DocPageView',
+            getProperties: ({ location, env }) => ({
+              path: location.path,
```

Comment:
```suggestion
              path: location.pathname,
```

Since this is a location object, `path` will be undefined